pytket/docs/changelog.rst

```diff
@@ -8,6 +8,14 @@ Major new features:
 
 * Support for ARM Linux platforms.
 
+Minor new features:
+
+* Updated implementation of ``ToffoliBox`` utilising multiplexors
+  for improved decomposition.
+* Add new ``DiagonalBox`` that synthesises a diagonal unitary matrix
+  into a sequence of multiplexed-Rz gates.
+* Add new ``MultiplexedTensoredU2Box`` that synthesises multiplexed tensor product of U2 gates.
```

Comment:
Maybe these all count as major new features? @cqc-alec what do you think?

Reply:
Agreed , major new features.